de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/DiskResourceMessages.properties

```diff
@@ -100,4 +100,5 @@ requestSuccess = Your request completed successfully!
 requestFailed = Unable to generate HT analysis path list file. Please try again!
 validationMessage = Please complete all the required fields!
 select = select...
+featureNotSupported=Feature not supported - invalid resource name
```

Comment:
When does this come up? I only see this line and the one in the corresponding java file, nothing that uses it. But I'm assuming something must.

Reply:
Ah, this is leftover.  I was going to have all the icons available but with this error message for any disk resources with bad names, but then I realized we disable everything and remove the icons in that case now so I went ahead and made it so that the dot menu wouldn't even render. I'll remove this.